plugins/catalog-backend/src/processing/DefaultCatalogProcessingEngine.ts

```diff
@@ -131,174 +133,194 @@ export class DefaultCatalogProcessingEngine implements CatalogProcessingEngine {
         }
       },
       processTask: async item => {
-        const track = this.tracker.processStart(item, this.logger);
+        await tracer.startActiveSpan('catalog_process_task', async span => {
```

Comment:
I couldn't see an obvious way of doing this cleanly - in particular for the automatic child span hierarchy to work, the work being done must be inside the callback function passed to `startActiveSpan`. Not sure how to achieve this if the callback function is being defined within the `tracker` object. (I'm not really a Node programmer)

This could be done with manual child span tracking, but then we'd lose any inner http & filesystem calls that are currently being picked up automatically (as they happen within the callback), which would be a pretty poor user experience